src/backend/base/langflow/api/v1/monitor.py

```diff
@@ -87,7 +87,7 @@ async def update_message(
     try:
         message_dict = message.model_dump(exclude_none=True)
         message_dict.pop("index", None)
-        monitor_service.update_message(message_id=message_id, **message_dict)
+        monitor_service.update_message(message_id=message_id, **message_dict)  # type: ignore
```

Comment:
`message_id` is passed as an `int` rather than string. we can update the `update_message` signature to expect an `int` instead